packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/billing-type/billing-type.controller.js

```diff
@@ -12,6 +12,10 @@ export default class KubernetesNodesBillingTypeCtrl {
   }
 
   switchToMonthlyBilling() {
+    this.trackClick(
+      'PublicCloud::pci::projects::project::kubernetes::details::nodepools::details::nodes::billing-type::confirm',
```

Comment:
I think its good to capture the billing mode in tracking like monthly or hourly. Something like
'PublicCloud::pci::projects::project::kubernetes::details::nodepools::details::nodes::billing-type::monthly::confirm'
'PublicCloud::pci::projects::project::kubernetes::details::nodepools::details::nodes::billing-type::hourly::confirm'